test/jdk/java/util/Formatter/Padding.java

```diff
@@ -26,7 +26,7 @@
  * @bug 4906370
  * @summary Tests to excercise padding on int and double values,
  *      with various flag combinations.
- * @run junit Padding
+ * @run junit/timeout=10 Padding
```

Comment:
/timeout=10 looks problematic, I don't think you want that.

Reply:
What is the issue with this? Is there a different way to set a timeout? The test tests that format does not take a long time to run so I would like to have a timeout

Reply:
> What is the issue with this? Is there a different way to set a timeout? The test tests that format does not take a long time to run so I would like to have a timeout

Hard-coded timeout in a test are generally considered harmful as the test suite is run on a wide variety of systems, a single value could be too large for fast systems and too small for slow ones. The jtreg harness has an overall `-timeout:N` factor which can scale up or down all the timeouts of individual tests. I don't know offhand if there is an existing idiom to do this with junit tests from within jtreg.

Reply:
Removed that timeout. Test fails before patch even when timeout factor is 10. Passes after patch within seconds. Default timeout should be good